packages/components/src/interfaces/complex/Accordion.ts

```diff
@@ -1,7 +1,16 @@
-import type { ProseContent } from "../native"
+import { Type, type Static } from "@sinclair/typebox"
+import { ProseSchema } from "../native"
 
-export interface AccordionProps {
-  type: "accordion"
-  summary: string
-  details: ProseContent
-}
+export const AccordionSchema = Type.Object(
+  {
+    type: Type.Literal("accordion"),
+    summary: Type.String({
+      title: "Accordion summary",
```

Comment:
this maps directly as a JSON Schema meta for [annotations](https://json-schema.org/understanding-json-schema/reference/annotations) right

Reply:
Yup, and these will be used directly as the form label and description inside JSONForms.